Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/PX1D-jiva-replica-node-affinity/replica-node-affinity

```diff
@@ -0,0 +1,187 @@
+bash Openshift-EE/utils/dump_cluster_state;
+#Update the e2e event for the job
+bash Openshift-EE/utils/event_updater jobname:jiva-replica-node-affinity $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+# Update the result of the test case in github mayadata-io/e2e-openshift repository.
+if [ "$?" != "0" ]; then
+python3 Openshift-EE/utils/result/result_update.py $job_id PX1D 4-chaos "Jiva replica node affinity" Fail $pipeline_id "$current_time" $commit_id $gittoken
+exit 1;
+fi
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase= busybox-deprovision-rep-node-affinity-jiva metadata="")
+echo $test_name
+
+################
+# LitmusBook 3 #
+################
+
+cd litmus
+# copy the content of deployer run_litmus_test.yml into a different file to update the test specific parameters.
+cp apps/busybox/deployers/run_litmus_test.yml deprovision_rep_node_affinity.yml
+
+# Update the environmental variables in litmus job spec.
+: << EOF
+  -----------------------------------------------------------------------------------------------------------------------------
+ | specAttribute     | kind   |         default value               | test specifc value                                       |
+  -----------------------------------------------------------------------------------------------------------------------------|
+ | appLabel          | env    | app=busybox-sts                     | app=busybox-rep-node-affinity-jiva                       | 
+ | appNamespace      | env    | app-busybox-ns                      | rep-node-affinity-jiva                                   |   
+ | storage clas      | env    | openebs-cstor-sparse                | openebs-jiva-default                                     |
+ | deploy type       | env    | statefulset                         | deployment                                               |
+ | Action            | env    | value: provision                    | value: deprovision                                       |
+ | Litmus Job name   | name   | generateName: litmus-busybox-deploy | generateName: busybox-deprovision-rep-node-affinity-jiva |
+ | Litmus job label  | label  | app: busybox-litmus                 | app: busybox-deprovision-rep-node-affinity               |
+  -----------------------------------------------------------------------------------------------------------------------------
+EOF
+
+sed -i -e 's/generateName: litmus-busybox-deploy/generateName: busybox-deprovision-rep-node-affinity-jiva /g' \
+-e 's/value: app-busybox-ns/value: rep-node-affinity-jiva/g' \
+-e 's/app: busybox-litmus/app:  busybox-deprovision-rep-node-affinity/g' \
+-e 's/value: statefulset/value: deployment/g' \
+-e 's/value: '\''app=busybox-sts'\''/value: '\''app=busybox-rep-node-affinity-jiva'\''/g' \
+-e 's/value: openebs-cstor-sparse/value: openebs-jiva-default/g' \
+-e 's/value: provision/value: deprovision/g' deprovision_rep_node_affinity.yml
```

Comment:
@shashank855 can you replace the same pvc name for deprovision also which is used to provision the busybox

Reply:
@nsathyaseelan I have made that change.